### PR TITLE
ADEN-11989 Include uppercase letters in the random string for beacon

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

File: src/shared/CookieManager.js [hunk 1]
```diff
@@ -14,17 +14,22 @@ class CookieManager {
...
         return resultValue;
     }
 
-    generateValue(name, withTimestamp) {
+    generateValue(withTimestamp) {
         // for users from GDPR countries if they did not give consent for tracking
         // we assign random values to session cookies; should match results of:
         // https://developer.fastly.com/reference/vcl/functions/randomness/randomstr/
-        let resultValue = (Math.random().toString(36) + '_________________').slice(2, 12)
+        const charsToPickFrom = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXTZabcdefghiklmnopqrstuvwxyz_';
```
Comment (preludek) [inline src/shared/CookieManager.js:27]: `validCharacters` maybe?

File: src/shared/CookieManager.js [hunk 2]
```diff
@@ -14,17 +14,22 @@ class CookieManager {
...
+
+        let resultValue = '';
+        for(resultValue; resultValue.length < 10;) {
```
Comment (preludek) [inline src/shared/CookieManager.js:30]: We dont need resultValue here - `for(; resultValue.length < 10;) {` would work the same. I'd prefer while loop here personally 🤷

File: src/shared/CookieManager.js [hunk 3]
```diff
@@ -14,17 +14,22 @@ class CookieManager {
...
+            resultValue += charsToPickFrom[(Math.random() * charsToPickFrom.length) | 0];
```
Comment (preludek) [inline src/shared/CookieManager.js:31]: I think I don't understand that `|  0` part 🙈. Why dont we just floor the random * charsToPickFrom.length?

**Review by preludek — APPROVED** [review]
Some nit picks

File: src/shared/CookieManager.js (hunk 3 above)
Comment (nandy-andy) [reply src/shared/CookieManager.js:31]: We need integers as the array's key are integers and `| 0` is shorter to write than `Math.floor()` 😈

**Pull request merged**